src/sentry/uptime/models.py

```diff
@@ -363,6 +375,24 @@ def create_detector_from_project_subscription(project_sub: ProjectUptimeSubscrip
         organization=project_sub.project.organization,
         source_id=str(project_sub.uptime_subscription_id),
     )
+    condition_group = DataConditionGroup.objects.create(
+        organization=project_sub.project.organization,
+    )
+    # TODO(epurkhiser): Should we be creating a new data condition + group for
+    # every uptime detector, or is there an intention to be able to re-use the
+    # groups for multiple detectors?
+    DataCondition.objects.create(
+        comparison=CHECKSTATUS_FAILURE,
+        type=Condition.EQUAL,
+        condition_result=DetectorPriorityLevel.HIGH,
```

Comment:
Should the result be a priority level? That seems to be what the stateful detector handler base class wants

Reply:
Yes - data conditions on a detector should return a priority level

🤔 I'm feeling like we should make an abstraction that has type hints for this based on if this is a detector trigger, workflow trigger, or action filter tho, would make this a bit easier and we can add docs to those individual areas then. #TODO